File: trunk/Java/OPSJLib/src/ops/Request.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package ops;

import configlib.ArchiverInOut;
import java.io.IOException;

/**
 *
 * @author Anton Gravestam
 */
public class Request extends OPSObject
{

    public String requestId = "";

    @Override
    public void serialize(ArchiverInOut archive) throws IOException
    {
        super.serialize(archive);
        requestId = archive.inout("requestId" , requestId);
    }



}

File: trunk/C#/Ops/Request.cs
///////////////////////////////////////////////////////////
//  Request.cs
//  Implementation of the Class Request
//  Created on:      12-nov-2011 09:25:35
//  Author:
///////////////////////////////////////////////////////////

namespace Ops 
{
	public class Request : OPSObject 
    {
		public string requestId = "";

		public override void Serialize(IArchiverInOut archive)
        {
            base.Serialize(archive);
            requestId = archive.Inout("requestId", requestId);
        }

	}

}